go.mod

```diff
@@ -157,13 +157,13 @@ require (
 	golang.org/x/text v0.3.5
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
 	golang.org/x/tools v0.1.0
-	google.golang.org/api v0.9.0
+	google.golang.org/api v0.13.0
```

Comment:
can you revert these other changes in go.mod and see if `make vendor_rebuild` and `go mod tidy` are happy?
(just change cobra)
this usually happens using as `go get` tries to upgrade other things for you (but i don't know why).

Reply:
Even if I just do s/0.0.5/1.1.3/ when vendor_rebuild.sh calls go mod vendor all these changes get re-made.